katas/metal-slug/daniel/Weapon.js

```diff
@@ -0,0 +1,34 @@
+export class Weapon {
+    constructor () {
+        this.name = '';
+        this.attackPower = 0;
```

Comment:
I think default could be `1`, a weapon with attack power `0` might not make too much sense